contracts/src/ModuleERC20.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.6.11;
+
+import "ds-token/token.sol";
+
+contract ModuleERC20 is DSToken  {
+    // sha256('cronos')[:20]
+    address constant module_address = 0x6526B48f897f6e08067dF00A1821d783cbC2af8b;
+
+    constructor(string memory denom, uint8 decimals_) DSToken(denom) public {
+        decimals = decimals_;
+    }
+
+    function mint_by_native(address addr, uint amount) public {
```

Comment:
`mint_by_module` , `burn_by_module`?

Not quite sure of the meaning of native

Reply:
native means native module, maybe let's keep the full name `mint_by_native_module`.

Reply:
or `mint_by_cosmos_module` to be more accurate? I think the word `native` have multiple meaning depending on the context. In the smart contract, native can be the current smart contract or some controllers

Reply:
cosmos_module or cronos_module? 😄

Reply:
hmm I dont have strong opinion on that, either is fine as long as we are consistent everywhere?

Reply:
renamed to `mint_by_cronos_module`.